lib/reviews.js

```diff
@@ -57,15 +57,23 @@ function parseFields($) {
 		var title = reviewContent.find('span[class=review-title]').text().trim();
 		var text = filterReviewText(reviewContent.text().trim(), title.length);
 
+		var developerComment = $(this).next().first();
```

Comment:
You're right. My point was to filter for class in next instead of comparing it as an attribute.